tests/test_run_data_managers.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# Stop pylint complaining from things that are necessary
+# for pytest to work.
+# pylint: disable=no-self-use
+import sys
+import time
+
+import pytest
+import yaml
+from docker_for_galaxy import start_container
```

Comment:
You can add `  # noqa: F401` at the end of this line and drop lines 17 and 18

Reply:
need the two spaces before the comment as usual, github formatting strips this here